app/mailers/diy_intake_email_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class DiyIntakeEmailMailer < ApplicationMailer
+
+  def high_support_message(diy_intake:)
```

Comment:
Could I18n.t() it with the diy_intake's locale, or pass the locale through into the job